### PR TITLE
Added support for Atrous Conv2D batch normalization folding

### PR DESCRIPTION
**Pull request by masadcv**
Added support for Atrous Conv2D batch normalization folding to handle the BatchToSpace and SpaceToBatch properly in tensorflow/contrib/quantize/python/fold_batch_norm.py

Also added unit tests for future development of bn folding for Atrous Conv2D in fold_batch_norm_test.py
The existing `_TestFoldAtrousConv2d` test was testing batch normalization folding for Depthwise Atrous Conv2D, hence renamed that test to a relevant name `_TestFoldDepthwiseAtrousConv2d.` All BN folding unit tests are passing locally on my machine.

### CODE REVIEW

**Comment by devkicks** [comment]
gentle ping for request to review this.

**Comment by mellanox-github** [comment]
Can one of the admins verify this patch?

**Comment by rthadur** [comment]
closing this PR as contrib folder will be depricated in 2.0, thank you.
CC @mihaimaruseac
